include/rsp.inc

```diff
@@ -263,44 +263,54 @@ makeNotImplemented tnei
     .endm
 .endm
 
-.macro makeLoadInstruction name, opcode
+.macro makeLoadInstruction name, opcode, offsetshift
     .macro \name vt, element, offset, base
         .ifgt (\element >> 4)
             .error "Invalid element"
         .else
-            .ifge (\offset)
-                .ifgt (\offset - 63)
-                    .error "Invalid offset - valid range: [-64, 63]"
-                .else
-                    loadVector \opcode, \vt, \element, \offset, \base
-                .endif
+            .ifne ((\offset) % (1 << \offsetshift))
+                .error "Invalid offset - must be multiple of operand size"
```

Comment:
This should possibly say the equivalent of "Invalid offset - must be a multiple of %d"